examples/notebooks/Splitters_Tutorial.ipynb

```diff
@@ -300,16 +300,16 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 85,
+   "execution_count": 6,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "([0, 1, 2, 3, 4, 5, 6, 8], [7, 9], [9])"
+       "([0, 1, 2, 3, 4, 5, 6, 8], [7], [9])"
```

Comment:
This looks a little strange to me. Why aren't the last two splits [8] and [9] instead of [7] and [9]. Potential bug?

Reply:
no it isn't, I have specified the valid index to be 7 that is why its showing 7, Which is what the indicesplitter is supposed to do, take the indices of valid and test data

Reply:
Ah, I see. Didn't read surrounding context carefully.